### DISCUSSION TITLE
can it add language model(such as BERT) as a backbone?

### DISCUSSION THREAD

**Discussion by wendilinplay** [RL-VIG/LibFewShot#37] [Q&A] [answered]
Hi, Just wondering if the fine-tune would work adding a pertained language model as a new backbone.

**Comment by WenbinLee** [comment, marked as answer]
Hi, this is an interesting question. However, because this framework is mainly built for the vision task, I am afraid it will be difficult to directly add a pre-trained language model. If you want to conduct a multi-modality model based on LibFewShot, it needs to take effort to modify the entire framework. Hope this can help you.
